pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -929,7 +929,37 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes).To(HaveLen(1))
 			Expect(causes[0].Field).To(Equal("fake.domain.memory.guest"))
 		})
-		It("should allow guest memory which is between requests and limits", func() {
+		It("should reject guest memory greater than requested memory when VM has VFIO device", func() {
```

Comment:
please add `should reject guest memory greater than requested memory when no VFIO device is requested`

Reply:
Should it be rejected in this case then? Currently there is no problems starting the VM when the guest memory is greater and there is no VFIO device requested; it only is failing to start and throwing the libvirt error if the VFIO device is present.

The current design is based off that behaviour, so it only will reject in the specified case if the VFIO device is present. WDYT?